lighthouse-extension/app/src/devtools-entry.js

```diff
@@ -45,27 +42,15 @@ function runLighthouseInWorker(port, url, flags, categoryIDs) {
   return lighthouse(url, flags, config, connection);
 }
 
-/**
- * Returns list of top-level categories from the default config.
- * @return {Array<{title: string, id: string}>}
- */
-function getDefaultCategories() {
-  const categories = Config.getCategories(defaultConfig);
-  categories.forEach(cat => cat.title = i18n.getFormatted(cat.title, 'en-US'));
-  return categories;
-}
-
 /** @param {(status: [string, string, string]) => void} listenCallback */
 function listenForStatus(listenCallback) {
   log.events.addListener('status', listenCallback);
 }
 
 if (typeof module !== 'undefined' && module.exports) {
-  // export for extension-entry to require (via browserify).
+  // export for require()ing (via browserify).
   module.exports = {
```

Comment:
since there's no unit tests for this bundle, it's not clear we actually want this `module.exports` block anymore. Do we?

Reply:
Yeah I'm not sure we need it either, but +1 to being paranoid for now :)